dashboard/config/scripts/levels/New Sprite Lab Project.level

```diff
@@ -30,9 +30,9 @@
     "hide_share_and_remix": "false",
     "disable_if_else_editing": "false",
     "free_play": "false",
-    "hide_animation_mode": "true",
+    "hide_animation_mode": "false",
```

Comment:
Do we need to do this across all the levels for which the curriculum team wants to have the costume tab?

Reply:
Already been added :)